src/docs/Components/Breadcrumb/code.mdx

```diff
@@ -0,0 +1,118 @@
+---
+title: 'Code'
+order: 2
+---
+
+## Basic implementation
+
+### Import
+
+Import **the settings** and **the breadcrumb** scss files.
+
+```scss
+@import 'settings-tools/_all-settings';
+
+@include import-font-families();
+
+@import 'components/_c.breadcrumb';
+```
+
+### Usage
+
+You can create breadcrumbs in the following way:
+
+- Use the `mc-breadcrumb` class on an HTML element `<nav>` as a wrapper
+- Apply the `mc-breadcrumb__container` class to the `<ul>` container of your breadcrumb thread
+- The `<li>` elements must have the class `mc-breadcrumb__item`
+- Finally each link must have the class `mc-breadcrumb__link`
+
+```html
+<nav class="mc-breadcrumb" aria-label="Breadcrumb">
+  <ul class="mc-breadcrumb__container">
+    <li class="mc-breadcrumb__item">
+      <a href="#" class="mc-breadcrumb__link">Level 00</a>
+    </li>
+    <li class="mc-breadcrumb__item">
+      <span class="mc-breadcrumb__link" aria-current="page">Level 01</span>
+    </li>
+  </ul>
+</nav>
+```
+
+<Preview path="default" />
```

Comment:
can't we use regular Mozaïc links to have the same behaviour (hover, focus, etc.)?

Reply:
That's a very good point.
I just made the breadcrumb links use the style sheet of the "Links" pattern.

I'll let you check it out.